tools/gource/make-video.sh

```diff
@@ -17,16 +17,20 @@ set -ex
 
 dst=$1
 shift
-$(dirname $0)/gource.sh \
-  --disable-progress    \
-  --stop-at-end         \
-  --output-ppm-stream - \
-  $@ |                  \
-avconv                  \
-  -y                    \
-  -r 60                 \
-  -f image2pipe         \
-  -vcodec ppm           \
-  -i -                  \
-  -vcodec libx264       \
+xvfb-run -a -s             \
```

Comment:
What is this, and how does it relate to this PR?

Reply:
It does not, I'll have to extract this :)